api/services/EmailService.js

```diff
@@ -46,6 +47,9 @@ function send(options, tpl, context) {
     message: options,
     locals: context,
   };
+  logger.info(
+    `[EmailService->send] About to send ${tpl} email to ${options.to}`,
+  );
   return email.send(args);
```

Comment:
then maybe the `logger.info` should follow successful transmission of the email. I see it's actually `return`ing the result of the email btu maybe it can be `await`ed and then returned after logging? if not, no biggie.

Reply:
It returns the promise back to the sendX function which itself returns the promise back to the controller, and the controller normally has a log message saying that the email was sent successfully.

Reply:
gotcha, thanks for the explanation